pkg/cloudprovider/vsphere/loadbalancer/interface.go

```diff
@@ -0,0 +1,119 @@
+/*
+ Copyright 2020 The Kubernetes Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package loadbalancer
+
+import (
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
```

Comment:
nit: in general can we keep imports grouped by the org name, i.e. `k8s.io` and `github.com` should be grouped separpately